### PR TITLE
7973-reset-password

### PR DESCRIPTION
**Pull request by leomendoza123**
## Required to also merge:  

https://github.com/ORCID/ORCID-Source/pull/6525
https://github.com/ORCID/orcid-ansible_inventory-private/pull/106
https://github.com/ORCID/orcid-reg-docker-private/pull/4

### CODE REVIEW

File: src/app/reset-password/reset-password/reset-password.component.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  styleUrls: [
+    './reset-password.component.scss',
+    './reset-password.component.scss-theme.scss',
+  ],
+  preserveWhitespaces: true,
+})
+export class ResetPasswordComponent implements OnInit {
+  form: FormGroup
+  hasNumberPatter = HAS_NUMBER
+  hasLetterOrSymbolPatter = HAS_LETTER_OR_SYMBOL
```
Comment (DanielPalafox) [inline src/app/reset-password/reset-password/reset-password.component.ts:28]: @leomendoza123 could you fix the typo in the two variables

File: src/app/core/password-recovery/password-recovery.service.ts [hunk 2]
```diff
@@ -48,4 +53,21 @@ export class PasswordRecoveryService {
         )
       )
   }
+
+  resetPasswordEmail(resetPassword: ResetPasswordEmailForm) {
+    let encodedData = JSON.stringify(resetPassword)
+    return this._http
+      .post<ResetPasswordEmailForm>(
+        environment.API_WEB + `reset-password-email-v2.json`,
```
Comment (DanielPalafox) [inline src/app/core/password-recovery/password-recovery.service.ts:61]: @leomendoza123 is this the correct endpoint to use?

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

File: src/app/core/password-recovery/password-recovery.service.ts (hunk 2 above)
Comment (leomendoza123) [reply src/app/core/password-recovery/password-recovery.service.ts:61]: yes

File: src/app/reset-password/reset-password/reset-password.component.html [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+  <mat-card class="mat-elevation-z3 orcid-popover">
+    <div class="popover-content orc-font-body-small">
+      <h2 class="orc-font-body-small" i18n="@@register.passwordRequirements">
+        Password requirements
+      </h2>
+      <p>
+        <ng-container i18n="@@register.passwordRequirements1"
+          >Passwords must be between be between</ng-container
+        >
+        <strong i18n="@@register.passwordRequirements2">8</strong>
```
[the remark this answers is not in the document]
Comment (leomendoza123) [reply src/app/reset-password/reset-password/reset-password.component.html:256]: We have done this before for the old app. I think we can't be sure all the future languages to support will use Arabian numbers.

**Review by DanielPalafox — APPROVED** [review]

**Pull request merged**